plugins/plugin-chart-echarts/src/utils/controls.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// eslint-disable-next-line import/prefer-default-export
+export function parseYAxisBound(bound?: string | number | null): number | undefined {
+  if (bound === undefined || bound === null || Number.isNaN(Number(bound))) {
```

Comment:
undefined returns NaN, so you don't technically need that, but if it helps for code-reading clarity, so be it :)

Reply:
Thanks for the review! Just to make sure I understood correctly: That last bit is to catch unparseable strings and `NaN`s (`undefined was already checked at the beginning). If this can be written more compactly I'm happy to learn+change!